sandcastle/api.py

```diff
@@ -321,9 +321,26 @@ def create_pod(self, pod_manifest: Dict) -> Dict:
 
         :return: response from the API server
         """
-        return self.api.create_namespaced_pod(
-            body=pod_manifest, namespace=self.k8s_namespace_name
-        )
+        idx = 1
+        # if we hit timebound quota, let's try 5 times with expo backoff
+        while idx < 6:
```

Comment:
Wouldn't it be clearer to use a for-loop here?

Reply:
And `for-else`?